test/e2e/velero_utils.go

```diff
@@ -32,7 +37,7 @@ func getProviderPlugins(providerName string) []string {
 	case "azure":
 		return []string{"velero/velero-plugin-for-microsoft-azure:v1.2.0"}
 	case "vsphere":
-		return []string{"velero/velero-plugin-for-aws:v1.2.0", "velero/velero-plugin-for-vsphere:v1.1.0"}
+		return []string{"velero/velero-plugin-for-aws:v1.2.0", "vsphereveleroplugin/velero-plugin-for-vsphere:1.1.0"}
```

Comment:
This needs to be reverted back to the original image name.

Reply:
That is the correct image name and source for the vSphere plug-in.  The old one didn't work.  https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/main/docs/vanilla.md